symphony-bdk-examples/bdk-template-examples/src/main/java/com/symphony/template/examples/FreeMarkerExample.java

```diff
@@ -0,0 +1,33 @@
+package com.symphony.template.examples;
+
+import com.symphony.bdk.template.api.Template;
+import com.symphony.bdk.template.api.TemplateException;
+import com.symphony.bdk.template.freemarker.FreeMarkerEngine;
+
+import java.util.HashMap;
+
+/**
+ * Sample class showing usage of FreeMarker templates.
+ */
+public class FreeMarkerExample {
+  public static void main(String[] args) throws TemplateException {
+    System.out.println(getStringFromBuiltInTemplate());
+    System.out.println(getStringFromCustomTemplate());
+  }
+
+  private static String getStringFromBuiltInTemplate() throws TemplateException {
+    Template template = new FreeMarkerEngine().newBuiltInTemplate("simpleMML");
```

Comment:
Do you think we need a enum listing all built-in templates we have? IMHO, it might be more convenient if we write something like: ```new FreeMarkerEngine().newBuiltInTemplate(BuiltInTemplate.SIMPLE_MML);```
WDYT?